eng/pipelines/templates/variables/globals.yml

```diff
@@ -9,3 +8,9 @@ variables:
   Codeql.SkipTaskAutoInjection: true
   # Disable warning until issue 21765 and 21766 are closed
   DisableDockerDetector: true
+
+  # Supported versions for testing. These variables are referenced in test matrix files and in pipeline yaml.
+  NODE_VERSION_N0: '22.x'
```

Comment:
Do the N0, N1, N2 have any significance? Will the versions they reference keep changing overtime?